options/glibc/include/resolv.h

```diff
@@ -12,6 +21,16 @@ int dn_expand(const unsigned char *, const unsigned char *,
 
 int res_query(const char *, int, int, unsigned char *, int);
 
+int res_init(void);
+
+/* From musl: Unused; purely for broken apps
+ * To avoid an massive struct, only add the items requested. */
+typedef struct __res_state {
+	unsigned long options;
+} *res_state;
+struct __res_state *__res_state(void);
+#define _res (*__res_state())
```

Comment:
Why is this needed?

Reply:
This is what musl does. As it states, this entire thing is purely for broken applications. It shouldn’t be used in new applications